.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.6"
+# command to install dependencies
```

Comment:
Nit: remove this and the other comment, as the yaml keys are self-explanatory.

Reply:
Yeah now that you say it, they are pretty redundant.